### PR TITLE
fix: Resolve BigInt type mismatch in eth_sendRawTransaction precheck

### LINKED ISSUE #3461: [hotfix] 0.65 seems to have a regression
### Problem

Relay 0.65 on mainnet seems to have a regression. 36% if the requests are failing with  `Cannot mix BigInt and other types, use explicit conversions`

### Solution

Investigate and resolve the problem. 

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by quiet-node**
**Description**:
This PR fixes the BigInt type mismatch in the eth_sendRawTransaction precheck. It also adds unit and acceptance tests to ensure the precheck handles null values for gasPrice, maxFeePerGas, and maxPriorityFeePerGas gracefully.

**Related issue(s)**:

Fixes #3461 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/relay/src/lib/precheck.ts [hunk 1]
```diff
@@ -221,8 +222,9 @@ export class Precheck {
       passes: false,
       error: predefined.INSUFFICIENT_ACCOUNT_BALANCE,
     };
-    const txGas = tx.gasPrice || tx.maxFeePerGas! + tx.maxPriorityFeePerGas!;
-    const txTotalValue = tx.value + txGas * tx.gasLimit;
+
+    const txGasPrice = BigInt(tx.gasPrice || tx.maxFeePerGas! + tx.maxPriorityFeePerGas!);
+    const txTotalValue = tx.value + txGasPrice * tx.gasLimit;
```
Comment (natanasow) [inline packages/relay/src/lib/precheck.ts:227]: q: `tx.value` and `tx.gasLimit` are already BigInt, right?

**Review by natanasow — APPROVED** [review]
LGTM, nice catch! :clap:

**Review by acuarica — APPROVED** [review]

File: packages/relay/src/lib/precheck.ts (hunk 1 above)
Comment (quiet-node) [reply packages/relay/src/lib/precheck.ts:227]: Yes they are all already bigint
